content/api/commands/intercept.md

```diff
@@ -2,18 +2,17 @@
 title: intercept
 ---
 
-Cypress routes all HTTP requests - including `XMLHttpRequest` (XHR) and `fetch` - through its proxy. Use `cy.intercept` to manage the behavior of these requests, including:
+Cypress routes all HTTP traffic initiated with `XMLHttpRequest` (XHR) and `fetch` through its proxy. `cy.intercept` can be used to capture or even change the behavior or contents of this traffic, including:
 
-<!-- TODO DX-182 add links to examples -->
-
-- Waiting on HTTP requests to complete before executing commands.
-- Making assertions and modifying (statically or dynamically):
-  - the request made by your front-end application.
-  - the response from your back-end service.
+- Waiting on HTTP request-response sessions to complete.
```

Comment:
This is my attempt at trying to delineate between a request (inclusive of the response) and just the request. Does that make sense?

Reply:
It does, but I'm not sure the distinction is necessary. Is there a concern that the reader won't be able to use contextual clues to differentiate between the "request is the whole thing" and "request object which is just a part of the whole thing"?

Reply:
In this case, I'm playing it safe. There's a chance someone could think that we're just waiting on the request to go out. However, if you believe this will cause more confusion I can reword it.